src/atoms/link/Link.js

```diff
@@ -1,2 +1,48 @@
 // @vue/component
-export default {}
+import getClass from '../../../utils/helpers/get-class.js'
+
+export default {
+  mixins: [ getClass ],
+  props: {
+    tag: {
+      type: String,
+      default: 'a'
+    }
+  },
+  data () {
+    return {
+      config: {
+        base: {
+          link: [
+            'no-underline',
+            'leading-relaxed',
+            'hover:underline',
+            'focus:underline',
+            'hover:cursor-pointer'
```

Comment:
cursor doesn't show as pointer for me. changing class to `cursor-pointer` seems to solve the issue